tests/virtualenv_tools_test.py

```diff
@@ -13,27 +13,41 @@ def auto_namedtuple(**kwargs):
     return collections.namedtuple('ns', tuple(kwargs))(**kwargs)
 
 
-@pytest.yield_fixture
-def venv(tmpdir):
+@pytest.yield_fixture(params=[None, '-e'])
```

Comment:
personally dislike `params` as I find it hard to read -- also this doubles the length of the test run

Please just install two modules instead -- it's more explicit and doesn't double the testsuite size

Reply:
The reason for installing non-editable is to test the rewriting of scripts copied verbatim into venv/bin/, and installing editable to test the rewriting of pth files. The editable flag affects everything referenced in the setup.py so I'm not sure how making two modules (mymodule1 and mymodule2, if I understand you correctly) would help?

Reply:
install one module editable, one module not

Reply:
or better yet, make a single test which installs a non-editable module into `venv`